frontend/packages/console-app/console-extensions.json

```diff
@@ -304,5 +304,28 @@
         "defaultValue": false
       }
     }
+  },
+  {
+    "type": "console.user-preference/item",
+    "properties": {
+      "id": "console.theme",
+      "label": "%console-app~Theme%",
+      "groupId": "general",
+      "description": "%console-app~Console will appear as per the selected theme.%",
+      "field": {
+        "type": "dropdown",
+        "userSettingsKey": "console.theme",
+        "defaultValue": "systemDefault",
+        "options": [
+          {
+            "value": "systemDefault",
+            "label": "%console-app~System default%"
+          },
+          { "value": "dark", "label": "%console-app~Dark%" },
+          { "value": "light", "label": "%console-app~Light%" }
```

Comment:
nit and personal perferences, but I would expect the order System default, Light, Dark :man_shrugging: